src/view.rs

```diff
@@ -94,94 +94,97 @@ use crate::{
     id::Id,
     style::{BoxShadowProp, Style, StyleClassRef},
     view_data::ViewStyleProps,
+    views::stack_from_iter,
     EventPropagation,
 };
 
 pub use crate::view_data::ViewData;
 
-pub type AnyWidget = Box<dyn Widget>;
+pub type AnyView = Box<dyn View>;
 
-pub trait View: Sized {
-    fn view_data(&self) -> &ViewData;
-    fn view_data_mut(&mut self) -> &mut ViewData;
+pub trait IntoAnyView {
+    fn any(self) -> AnyView;
+}
 
-    fn id(&self) -> Id {
-        self.view_data().id()
+impl<V: View + 'static> IntoAnyView for V {
+    fn any(self) -> AnyView {
+        Box::new(self)
     }
+}
 
-    /// This method builds the widget described by the view. Implementations may not assume that an
-    /// implicit scope is available. It may differ from the scope the view was created in or may not
-    /// be available at all.
-    fn build(self) -> AnyWidget;
+pub trait IntoView {
+    fn data(&self) -> &ViewData;
 
-    /// Converts this view into the `AnyView` type.
-    fn any(self) -> AnyView
-    where
-        Self: 'static,
-    {
-        AnyView {
-            view: Box::new(self),
-        }
-    }
-}
+    fn data_mut(&mut self) -> &mut ViewData;
 
-/// This is a type which can hold any view.
-pub struct AnyView {
-    view: Box<dyn DynView>,
+    fn into_view(self) -> impl View;
```

Comment:
You should keep the note on scopes as this seems to be a rename of the current `View::build`.